m4_sessions/1806-inning/group8.md

```diff
@@ -1,22 +1,32 @@
 ## Group Members:
-
+Margaret, Clee, Camilo, Daniela
 
 
 ## Name of Session: 
+All These Flavors and You Chose to be Salty
 
 ## Session Description 
-
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+The benefits of being positive 
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+* tools for changing your mindset from negative to positive
+* Action item: change mindset about something they are negative about. 
 
 ## Session Outline 
 
 (Please include who is facilitating and how much time you are going to spend in each section):
-
-
+Introduce the session and goals for the day
+Intros - Name, Mod, Why did you come to Turing
+Line up in the room from 'most positive' to 'least positive'
+     Pair with someone opposite side of the room - discuss things at Turing that make you negative and why 
+     Share Out 
+Group presenting
```

Comment:
I'd like to see slides on this, esp with some of the resources you already have.  How can you divide this up so each of you takes a slide and gets involved?